README.md

```diff
@@ -425,6 +425,22 @@ Sometime it is required to load sensitive configurations (GitHub/Snyk's token) f
 * Change the workdir of the docker image to be `/broker`/
 Example of such file is located in your broker container at $HOME/.env
 
+### Troubleshooting
+
+#### Support of big manifest files (> 1Mb) for GitHub / GitHub Enterprise
+
+One of the reason for failing of open Fix/Upgrade PRs or PR/recurring tests might be fetching big manifest files (> 1Mb) failure. To address this issue, additional GraphQL query should be whitelisted in `accept.json`:
+
+- should be under `query graphql` section in `valid` array
+```json
+{
+    "//": "query to get file SHA",
+    "path": "query",
+    "regex": "{\\s*repository\\(\\s*owner:\\s*\"[a-zA-Z0-9-_.]+\",\\s*name:\\s*\"[a-zA-Z0-9-_.]+\"\\)\\s*{\\s*object\\(\\s*expression:\\s*\"[a-zA-Z0-9-_./]+:[a-zA-Z0-9-_./]+\"\\)\\s*{\\s*\\.\\.\\.\\s*on\\s*Blob\\s*{\\s*oid\\,\\s*}\\s*}\\s*}\\s*}\\s*"
+}
+```
+**Note**: this endpoint returns file's SHA, that gives Snyk access to all files in given repository.
```

Comment:
I meant the file sha endpoint is actually safe, we already have the find globs endpoint that returns shas and that is not a security issue.

The security problem is the blobs API (the one that given a sha returns the file contents).

So we should add the file sha to the default rules and put the blobs rule in this document as an optional one.

Reply:
Oh yes, correct